### DISCUSSION TITLE
Display speed while not routing

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#500] [Q&A] [answered]
Hi Emux, can you make it possible to display the speed while not using guided routing, just the map?
And - If you add the possibility to show the speed limit permanently (other tread), this as well?
Thanks, cheers Dirk

**Comment by devemux86** [comment, marked as answer]
Current speed should already be available in all modes: navigation, follow location, browse map.
Can try it in the latest Beta in "Settings | Application | Display":
- https://github.com/devemux86/cruiser/discussions/482

Speed limits are currently provided by the calculated route, so they are available in navigation.
An alternative way is in my list to check:
- https://github.com/devemux86/cruiser/discussions/452

**Comment by DIrgi** [comment]
1st of all I'm using the latest beat version 3.043-b2
Speed is only available for: Navigation & Follow location - what make sense means..browsing the map, I need no "speed".
However I recently used Cruiser without navigation just map display while driving. Afaik speed was not shown although "Follow location" was activated (Navigation anyway). I will cross-check and come back when I use cruiser again.
Only altitude and satellites where shown while driving without navigation. 

2nd: I checked your link and discussion for speed limit. There was another discussion recently with the request to have (on demand) a continuously speed limit sign available all the time like this:
![image](https://github.com/devemux86/cruiser/assets/57968526/15e8fa66-6a9e-4ba8-8b33-18aeac9c02fa)
That was my link to the discussion I meant.
However, I asked to have this speed limit sign as well available in the "Following location" process and not only the "Navigation" mode.
As long as Cruiser knows where I'm driving on the map via GPS, Cruiser knows which speed limit is available on the road I'm driving - or not?

**Reply by devemux86** [reply]
> 3.043-b2

Please try the latest Beta, it is now b9.

Follow location is not navigation and can be activated with long press the location button.
Then need to enable the speed in application display settings for the follow location mode.

> As long as Cruiser knows where I'm driving on the map via GPS, Cruiser knows which speed limit is available on the road I'm driving - or not?

It does not work like that, as mentioned above, speed limits are provided by the calculated route.
And only navigation parses and uses the route. Follow location just centers map on GPS location.

So anything more requires a different provider for speed limits, we will check it in the future.
